stock_available_mrp/tests/test_potential_qty.py

```diff
@@ -12,6 +12,14 @@ class TestPotentialQty(TransactionCase):
     def setUp(self):
         super(TestPotentialQty, self).setUp()
 
+        self.product_model = self.env["product.product"]
+        self.bom_model = self.env["mrp.bom"]
+        self.bom_line_model = self.env["mrp.bom.line"]
+        self.stock_quant_model = self.env["stock.quant"]
+
+        self.setup_demo_data()
+
+    def setup_demo_data(self):
```

Comment:
Would `SavepointCase` help you with this? You'd set up the demo data once for all at class setup and you can get the models in instance setup.
(not sure if it's more readable to put models here, but I won't object).